docs/what-is-asciidoc.md

```diff
@@ -102,18 +102,21 @@ If you're not already using one of these text editors of IDEs, don't feel the ne
 It's possible that your existing text editor or IDE already has an AsciiDoc plugin.
 And it makes far more sense to continue using what you already know, instead of learning an entirely new tool.
 
-## Previewing AsciiDoc Changes
+## An AsciiDoc Previewer
 
 ![Viewing AsciiDoc in Firefox with the Asciidoc Live Preview Plugin](./images/viewing-asciidoc-in-firefox-with-the-asciidoc-live-preview-plugin.png)
 
-Once your text editor or IDE is ready to work with AsciiDoc then, assuming it doesn’t already support live preview, you’ll need a way to preview the changes you’re making to ensure that they render as you expect.
+Once your text editor or IDE is ready to work with AsciiDoc then, assuming it doesn't already support previewing AsciiDoc, you’ll need a way to preview the changes you’re making to ensure that they render as you expect.
 
-For that, assuming that you’ve installed _Google Chrome_, _Mozilla Firefox_, or _Opera_, you need to install the AsciiDoc Live Preview plugin.
-When installed, you can toggle AsciiDoc preview, and see AsciiDoc files rendered in the browser.
+For that, assuming that you’re using one of _Google Chrome_, _Mozilla Firefox_, or _Opera_, you need to install the [AsciiDoc Live Preview plugin](https://asciidoctor.org/docs/editing-asciidoc-with-live-preview/).
+When installed, enable the AsciiDoc Live Preview plugin, and you will see AsciiDoc files rendered as HTML in the browser.
 
-You can install the extension through the browser or find it through the links below.
+## AsciiDoc Documentation
 
-- [Google Chrome plugin](https://chrome.google.com/webstore/detail/asciidoctorjs-live-previe/iaalpfgpbocpdfblpnhhgllgbdbchmia?hl=en)
-- [Mozilla Firefox add-on](https://addons.mozilla.org/en-US/firefox/addon/asciidoctorjs-live-preview/)
-- [Opera plugin](https://addons.opera.com/en/extensions/details/asciidoctorjs-live-preview/)
+As with anything of a technical nature, you need documentation, because there’s going to come a time that you will need to refer to it.
+So to with AsciiDoc.
```

Comment:
Need a language advice...
``So to with...`` or ``So too with...``

Reply:
Good spotting.